packages/style/components/queryBuilder/Combiner.module.scss

```diff
@@ -7,6 +7,19 @@
         display: flex;
         align-items: center;
         height: auto;
+        border: none;
+
+        &:hover {
+            border: none;
+            background: transparent;
+            color: $query-combiner-button-hover-color;
+        }
+        &:focus {
+            border: none;
+            background: transparent;
+            color: $query-combiner-button-focus-color;
+        }
+
```

Comment:
mixins with 1 variable to reuse in all places?

Reply:
there is already a variable you can use to match them all, but I let the customization if anyone wants to have different style but still match the graphical charts. (lets say copy has a hover blue but the trash can is red)